profitbricks/src/main/java/org/jclouds/profitbricks/domain/Server.java

```diff
@@ -84,14 +85,20 @@ public static Status fromValue(String v) {
    @Nullable
    public abstract List<Nic> nics();
 
+   @Nullable
+   public abstract String balancedNicId();
+
+   @Nullable
+   public abstract boolean activate();
```

Comment:
Now that I see it among the properties, it kinda looks weird. Maybe we change this instead into `balanced`? (`server.balanced()` vs `server.activate()`)